src/AndcultureCode.CSharp.Sitefinity.Testing/Extensions/IRestResponseExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using AndcultureCode.CSharp.Sitefinity.Testing.Constants;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using Shouldly;
+using System.Net;
+
+namespace AndcultureCode.CSharp.Sitefinity.Testing.Extensions
+{
+    public static class IRestResponseExtensions
+    {
+        /// <summary>
+        /// Checks if the response returned from Sitefinity is the standard
+        /// "Please wait a moment" response, indicating that the site is still
+        /// starting up.
+        /// </summary>
+        /// <param name="restResponse"></param>
+        /// <returns>True if the response indicates Sitefinity is still starting up.</returns>
+        public static bool IsWaitAMomentResponse(this IRestResponse restResponse)
```

Comment:
Suggestion: Since this, and the one after are single line functions, consider converting to expression bodies.